source/digits_hits/include/GateRejectedVolume.hh

```diff
@@ -0,0 +1,64 @@
+/*----------------------
+   Copyright (C): OpenGATE Collaboration
+
+This software is distributed under the terms
+of the GNU Lesser General  Public Licence (LGPL)
+See LICENSE.md for further details
+----------------------*/
+
+
+#ifndef GateRejectedVolume_h
+#define GateRejectedVolume_h 1
+
+#include "globals.hh"
+#include <iostream>
+
+#include "GateVPulseProcessor.hh"
+
+class GateRejectedVolumeMessenger;
+
+
+/*! \class  GateRejectedVolume
+    \brief  Pulse-processor modelling a simple volume discriminator.
+
+    - The method ProcessOnePulse of this class models a simple
+      volume discriminator: any input pulse whose volume name is not as
+      defined is copied into the output pulse-list.
+      On the contrary, any input pulse whose name is the same as 
+      specified is discarded.
+
+      \sa GateVPulseProcessor
+*/
+class GateRejectedVolume : public GateVPulseProcessor
+{
+  public:
+
+    //! Constructs a new module attached to a GateDigitizer
+    GateRejectedVolume(GatePulseProcessorChain* itsChain, const G4String& itsName, G4String rejectedVolume);
+    //! Destructor
+    virtual ~GateRejectedVolume() ;
+
+    //! Returns the rejected volume
+    G4String GetRejectedVolume()   	    { return m_rejectedVolume; }
+
+    //! Set the threshold
+    void SetRejectedVolume(G4String val)   { m_rejectedVolume = val; }
```

Comment:
Looks like still not adjusted to the same level (``GetRejectedVolume`` have 1 less space then ``SetRejectedVolume``). If you think, that adjusting them to same level looks wrong, you can also just change this to:
```
G4String GetRejectedVolume() { return m_rejectedVolume; }

//! Set the threshold
void SetRejectedVolume(G4String val) { m_rejectedVolume = val; }
```
I think, it looks ok too.